### DISCUSSION TITLE
Can this be client side only?

### DISCUSSION THREAD

**Discussion by mcclellanja** [Sinytra/Connector#815] [Q&A] [answered]
Does this need to be server side as well as client side if I am only using client side only fabric mods in my mod pack? I am only using this for the replay mod currently and that runs completely client side, and a few other players on the server don't care about replay mod and I was wondering if I don't install it on the server will I still be able to connect?

**Comment by coco875** [comment, marked as answer]
Yes

**Reply by coco875** [reply]
If the fabric mod work also like this
